### ISSUE TITLE
模型训练咨询

### ISSUE THREAD

**Issue by 642721838** [PaddlePaddle/PaddleSeg#649]
你好，我在使用paddleseg里面的预训练模型unet_bn_coco进行3波段数据的训练，结果出来这个警告，请问对模型结果有影响么，另：我的损失函数收敛比较慢，但是bathsize根据GPU内存已经设置为最大了。为4，请问怎么解决让loss收敛的更快一些呢，谢谢！
W1204 08:58:27.482141  2328 device_context.cc:252] Please NOTE: device: 0, CUDA Capability: 75, Driver API Version: 11.0, Runtime API Version: 10.0
W1204 08:58:27.576633  2328 device_context.cc:260] device: 0, cuDNN Version: 7.6.
[SKIP] Shape of pretrained weight ./pretrained_model/unet_bn_coco//logit/weights doesn't match.(Pretrained: (21, 64, 3, 3), Actual: (2, 64, 3, 3))
2020-12-04 08:58:32,505-WARNING: ./pretrained_model/unet_bn_coco/.pdparams not found, try to load model file saved with [ save_params, save_persistables, save_vars ]
There are 90/91 varaibles in ./pretrained_model/unet_bn_coco/ are loaded.
Use multi-thread reader
W1204 08:58:33.484915  2328 build_strategy.cc:170] fusion_group is not enabled for Windows/MacOS now, and only effective when running with CUDA GPU.
如图：
![image](https://user-images.githubusercontent.com/46669502/101116915-f1e37080-3620-11eb-9500-2b054ed7f740.png)

**Comment by wuyefeilin** [comment]
你好，运行是正常的，这是正常警告。

你所谓的收敛慢是怎么样一个趋势呢？ 你可以打开vdl，然后截一张loss的曲线图让我们看看。

**Comment by 642721838** [comment]
你好，我之前是初始学习率0.001、200个epoch，loss下降较慢。后来把初始学习率降低了（0.0001），然后就收敛很快，epoch在90的时候loss就收敛了，而且值比200的时候要低0.01-0.02左右。




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "PaddlePaddle/PaddleSeg"                                                                                    <notifications@github.com&gt;;
发送时间:&nbsp;2020年12月7日(星期一) 下午3:01
收件人:&nbsp;"PaddlePaddle/PaddleSeg"<PaddleSeg@noreply.github.com&gt;;
抄送:&nbsp;"棉花糖"<642721838@qq.com&gt;;"Author"<author@noreply.github.com&gt;;
主题:&nbsp;Re: [PaddlePaddle/PaddleSeg] 模型训练咨询 (#649)





 
你好，运行是正常的，这是正常警告。
 
你所谓的收敛慢是怎么样一个趋势呢？ 你可以打开vdl，然后截一张loss的曲线图让我们看看。

**Comment by wuyefeilin** [comment]
可以调节学习率， weight decay， batch size等超参数，或者更换优化器等等
